database/oracle/setupDatabase.sh

```
@@ -0,0 +1,3 @@
+#!/bin/bash
+echo "Setting up the Oracle Database..."
+$ORACLE_HOME/bin/dbca -silent -createDatabase -responseFile /opt/oracle/dbca.rsp
```

Shellcheck

[info] Double quote to prevent globbing and word splitting.

(SC2086)